### ISSUE TITLE
it can not resize on the left side after the width of element is minimum size(5)

### ISSUE THREAD

**Issue by daixiaoju** [taye/interact.js#665]
it can not resize on the left side after the width of element is minimum restrict size(min:5)

this is the code:
```
interact('.shift-layer')
				.resizable({
					allowFrom: '.selected',
					edges: { left: true, right: true },
					restrictSize: {
						min: { width: 5 }
					}
				})
```

### Expected behavior
it should be able to resize it on both sides.

### Actual behavior
it can only resize on the left side after it shorten to the minimum size.

### System configuration
**interact.js version**: 1.3.4
**Browser name and version**:  Chrome, 68.0.3440.106
**Operating System**: Window 10

**Comment by daixiaoju** [comment]
@taye , we really need your help, thanks!

**Comment by taye** [comment]
Please make a demo on https://jsfiddle.net/

**Comment by daixiaoju** [comment]
Hi, i have already created a demo for it, please help to check! http://jsfiddle.net/tzs8p94a/

**Comment by taye** [comment]
I've published version `1.4.0-alpha.16` which has a fix for this.
